AutoCollection/Statsbeat.ts

```diff
@@ -18,17 +19,7 @@ class Statsbeat {
 
     private static TAG = "Statsbeat";
 
-    // Counters
-    private _lastRequests: { totalRequestCount: number; time: number; };
-    private _totalRequestCount: number = 0;
-    private _totalSuccesfulRequestCount: number = 0;
-    private _totalFailedRequestCount: number = 0;
-    private _retryCount: number = 0;
-    private _exceptionCount: number = 0;
-    private _throttleCount: number = 0;
-    private _intervalRequestExecutionTime: number = 0;
-    private _lastIntervalRequestExecutionTime: number = 0;
-
+    private _networkStatsbeatCollection: Array<Network.NetworkStatsbeat>;
```

Comment:
I'm wondering if a dictionary can be used here? With (category, endpoint) as the key?

Reply:
Normal size of this array would be less than 3, could be slightly bigger in case of redirects triggered from the backend, using some kind of hash table sound to much for this scenario